### PR TITLE
docs: update for k8s support for igw and header manip

### PR DESCRIPTION
**Pull request by lkysow**
Add docs now that k8s supports these new config entry fields

### CODE REVIEW

File: website/content/docs/connect/config-entries/ingress-gateway.mdx [hunk 1]
```diff
@@ -981,21 +991,25 @@ You can specify the following parameters to configure ingress gateway configurat
         },
         {
           name: 'TLSMinVersion',
+          yaml: false,
```
Comment (lkysow) [inline website/content/docs/connect/config-entries/ingress-gateway.mdx:994]: we don't support these in k8s right now

**Review by thisisnotashwin — APPROVED** [review]
💥

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/575394.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit ecc5dae06ff99f58ed200604dba33f097a09562c onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit ecc5dae06ff99f58ed200604dba33f097a09562c onto `release/1.11.x` succeeded!

**Pull request merged**